dockerfiles/docker-compose.yml

```diff
@@ -35,17 +35,15 @@ services:
   wrangler:
     build:
       context: ${PWD}
-      dockerfile: ${PWD}/common/dockerfiles/Dockerfile.wrangler
+      dockerfile: ${PWD}/common/packages/addons-inject/Dockerfile
     volumes:
-      - ${PWD}/common/dockerfiles/force-readthedocs-addons.js:/usr/src/app/docker/force-readthedocs-addons.js
+      - ${PWD}/common/packages/addons-inject/:/usr/src/app/packages/addons-inject/
     networks:
       readthedocs:
+    working_dir: /usr/src/app/packages/addons-inject/
     command: [
       "wrangler",
       "dev",
-      "/usr/src/app/docker/force-readthedocs-addons.js",
-      "--name=force-addons",
-      "--latest",
```

Comment:
Auto reload does work, it actually works a bit better now with the directory mounted. Mounting just the file required restarting the wrangler container for me originally, because the file would disappear in the container on save. Wrangler detects the file update successfully now.